shenyu-common/pom.xml

```diff
@@ -69,5 +75,44 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-all</artifactId>
+            <version>${grpc.version}</version>
+        </dependency>
+
```

Comment:
Why do you put these dependencies in `common` module?

Reply:
Because two objects generated by grpc is needed on both the client and server.  one object as request messge, one object as response messge.

Reply:
I think about how to improve this question, thanks~

Reply:
HI, i'm so sorry, the current optimization method does not remove this dependency. Next, I will try to optimize in other ways.